apps/sitemaps/routes.coffee

```diff
@@ -112,7 +112,15 @@ getArtworkBuckets = (callback) ->
       limit: FUSION_PAGE_SIZE
       'fields[]': switch template
         when 'artworks' then ['id']
-        when 'images' then ['id', 'images', 'title', 'artist', 'date']
+        when 'images' then [
+          'id'
+          'images.image_urls.large'
+          'images.image_urls.larger'
+          'images.is_default'
```

Comment:
Fusion has the ability to limit the fields being request using `&fields[]=images.image_urls...` params. This was before we dived into GraphQL in MP—so not surprising to see the overlap in use case 😛 . The bigger ticket here is to drop Fusion in favor of MP, but this was low hanging fruit to plug a hole.